package.json

```diff
@@ -13,7 +13,8 @@
     "@google-cloud/compute": "^3.1.1",
     "@google-cloud/resource-manager": "^3.0.0",
     "google-auth-library": "^7.11.0",
-    "googleapis": "^95.0.0"
```

Comment:
Please remove caret from all the versions, to avoid suprises when the author releases a faulty package which would match the version range.